code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -463,14 +463,35 @@
 	fire_sound = 'sound/items/bikehorn.ogg'
 	projectiles = 10
 	projectile_energy_cost = 500
+	harmful = TRUE
 	diags_first = TRUE
+	var/punch_damage = 35 ///damage done by the glove on contact
+	var/can_toggle_lethal = TRUE ///TRUE - can toggled between lethal and non lethal || FALSE - cannot toggle
 
 /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/launcher/punching_glove/can_attach(obj/mecha/combat/honker/M)
 	if(..())
 		if(istype(M))
 			return 1
 	return 0
 
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/launcher/punching_glove/get_equip_info()
+	if(!chassis)
+		return
+
+	if(can_toggle_lethal)
+		return "[..()] &nbsp; <a href='?src=[REF(src)];lethalPunch=1'>[harmful?"Punch":"Pat"] mode</a>"
+	else
+		return
+
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/launcher/punching_glove/Topic(href, href_list)
+	..()
+	if(href_list["lethalPunch"])
+		harmful = !harmful
+		if(harmful)
+			chassis?.occupant_message("<span class='warning'>Lethal Fisting Enabled.</span>")

```

Comment:
while funny, our players cannot be trusted with fisting
https://github.com/tgstation/tgstation/pull/40941

Reply:
what a clown does in the privacy of his own HONK is none of my business 😳 (it only shows up to the mech pilot so it's probably fine)